odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/repository/TagRepositoryImplTest.java

```diff
@@ -1,17 +1,217 @@
 package org.opendatadiscovery.oddplatform.repository;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.jetbrains.annotations.NotNull;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.opendatadiscovery.oddplatform.BaseIntegrationTest;
+import org.opendatadiscovery.oddplatform.dto.TagDto;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.DataEntityPojo;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.TagPojo;
+import org.opendatadiscovery.oddplatform.utils.Page;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 @DisplayName("Integration tests for TagRepository")
 class TagRepositoryImplTest extends BaseIntegrationTest {
 
     @Autowired
     private TagRepository tagRepository;
+    @Autowired
+    private DataEntityRepository dataEntityRepository;
+
+    @Test
+    @DisplayName("Creates tag pojo, expecting tag pojo in db")
+    void testCreateTagPojo() {
+        final TagPojo tag = new TagPojo()
+            .setName(UUID.randomUUID().toString())
+            .setImportant(true);
+
+        final TagPojo expectedTagPojo = tagRepository.create(tag);
+
+        assertThat(expectedTagPojo).isNotNull();
+        assertThat(expectedTagPojo.getName()).isNotNull();
```

Comment:
here, instead of not null check, you can check that expectedTagPojo.getName == tag.getName and expectedTagPojo.getImportant == tag.getImportant